src/monocle_apptrace/utils.py

```diff
@@ -118,11 +138,12 @@ def get_wrapper_method(package_name: str, method_name: str):
     wrapper_module = import_module("monocle_apptrace." + package_name)
     return getattr(wrapper_module, method_name)
 
+
 def update_span_with_infra_name(span: Span, span_key: str):
```

Comment:
This code should not be needed right ?

Reply:
We should move this to metamodel output processor for root span

Reply:
removed, not required.